index.txt

```diff
@@ -82,13 +82,11 @@ install_caddy()
 	fi
 
 	uname="$(uname)"
-	if grep -qi 'Darwin' <<<"$uname"; then
+	if [[ ${uname^^} == *DARWIN* ]]; then
```

Comment:
Because you used grep -i (case insensitive), ${X^^} converts all lowercase in $X to uppercase. :-)